rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -762,6 +762,8 @@ def _decode_transaction(
                     continue  # since the input data rule found events for this log
 
             decoding_output = self.decode_by_address_rules(context)
+            if decoding_output.stop_processing is True:
+                return [], False, None
```

Comment:
Just a comment explaining what/why

```suggestion
                return [], False, None  # We determined this transaction is full of unnecessary log events and should all be skipped. Saves processing time.
```